nebula-unity/Assets/Scripts/Bullet.cs

```diff
@@ -0,0 +1,31 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Bullet : MonoBehaviour
+{
+    // Start is called before the first frame update
+    float damage;
+
+    void Start()
+    {
+        damage = 25f;
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+        
+    }
+
+    void OnBecameInvisible(){
```

Comment:
Be careful with MonoBehaviour.OnBecameInvisible().  This is specifically triggered when the object is not "seen" by any cameras and we need to make sure we don't allow any camera shenanigans that will despawn bullets in ways we don't want.